applications/portal/templates/statefulset.yaml

```diff
@@ -181,3 +211,15 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+  {{- if .Values.config.volumes.privateWorkarea.pvc }}
+  volumeClaimTemplates:
+    - metadata:
+        name: "firefly-private-workarea"
```

Comment:
The `StatefulSet` pod name will be appended to this name, so you want to set it to something way shorter, like `workarea`. It will be obvious that it's private since the pod name will be part of the name.